src/shared-components/DataTable/hooks/useTableData.spec.tsx

```
@@ -0,0 +1,1982 @@
+      fetchMore: mockFetchMore,
+      networkStatus: NetworkStatus.ready,
+    } as unknown as QueryResult<unknown>;
+
+    const { getByText } = render(
+      React.createElement(() => {
+        const { refetch, fetchMore } = useTableData(result1, {
+          path: ['users'],
+        });
+        return (
+          <div>
+            <button onClick={() => refetch()}>Refetch</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)